.eslintrc.yml

```diff
@@ -2,3 +2,15 @@ extends: ["eslint:recommended", "google"]
 env:
   es6: true
   browser: true
+"parserOptions": {
+  "ecmaVersion": 2017
+}
+"rules": {
+  "comma-dangle": ["error", {
+    "arrays": "always",
+    "objects": "always",
+    "imports": "never",
+    "exports": "never",
+    "functions": "never"
+  }]
+}
```

Comment:
Use YAML syntax instead of JSON (or refactor the whole file and change the extension)